x-pack/plugins/fleet/common/mocks.ts

```diff
@@ -56,3 +57,27 @@ export const deletePackagePolicyMock = (): DeletePackagePoliciesResponse => {
     },
   ];
 };
+
+/**
+ * Creates mock `authz` object
+ */
+export const createFleetAuthzMock = (): FleetAuthz => {
```

Comment:
Moved it from `server/mocks` so that it can be reused from `public/` code as well